packages/strapi/bin/strapi-new.js

```diff
@@ -39,6 +42,20 @@ module.exports = function (name, cliArguments) {
     developerMode
   };
 
+  if (_.values(_.omit(cliArguments, ['dev'])).length) {
+    scope.database = {
+      settings: {
```

Comment:
Two optional arguments are missing `schema` and `socket` for Postgres and MySQL.

Reply:
For MySQL, do you talk about [that socketPath property](https://github.com/sjelfull-forks/knex/commit/ebfdfd92c4687f636673daf35e5086111bad9e4b#diff-eacf331f0ffc35d4b482f1d15a887d3bR426) ?
And for the Postgres, I didn't find the schema property event in https://github.com/strapi/strapi/blob/master/packages/strapi-bookshelf/lib/utils/connectivity.js

Reply:
It seems that both properties are not implemented in the interactive CLI but we should... Especially the schema for PG...

https://github.com/strapi/strapi/blob/master/packages/strapi-knex/lib/index.js#L98

Reply:
Can it be another PR to implement that?
It is a special use case that shouldn’t be commit in this PR.

Reply:
Not so special for PG users but I agree, we could do that later.